src/Device/device.jsx

```diff
@@ -0,0 +1,132 @@
+ * MERCHANTABILITY, NONINFRINGEMENT, AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL NORDIC SEMICONDUCTOR ASA OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
+ * TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+ * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+const deviceInfo = pcaNumber => ({
+    PCA10028: {
+        name: 'nRF51 DK',
+        cores: 1,
+        website: {
+            productPagePath: 'Development-Kits/nRF51-DK',
+            buyOnlineParams: 'search_token=nrf51-DK&series_token=nRF51822',
+        },
+    },
+    PCA10031: {
+        name: 'nRF51 Dongle',
+        cores: 1,
+        website: {
+            productPagePath: 'Development-Kits/nRF51-Dongle',
+            buyOnlineParams: 'search_token=nRF51-Dongle&series_token=nRF51822',
+        },
+    },
+    PCA10040: {
+        name: 'nRF52 DK',
+        cores: 1,
+        website: {
+            productPagePath: 'Development-Kits/nRF52-DK',
+            buyOnlineParams: 'search_token=nRF52-DK&series_token=nRF52832',
+        },
+    },
+    PCA10056: {
+        name: 'nRF52840 DK',
+        cores: 1,
+        website: {
+            productPagePath: 'Development-Kits/nRF52840-DK',
+            buyOnlineParams: 'search_token=nrf52840-DK&series_token=nRF52840',
+        },
+    },
+    PCA10059: {
+        name: 'nRF52840 Dongle',
+        cores: 1,
+        website: {
+            productPagePath: 'Development-Kits/nRF52840-Dongle',
+            buyOnlineParams: 'search_token=nRF52840DONGLE&series_token=nRF52840',
+        },
+    },
+    PCA10090: {
+        name: 'nRF9160 DK',
+        cores: 1,
+        website: {
+            productPagePath: 'Development-Kits/nRF9160-DK',
+            buyOnlineParams: 'search_token=nrf9160-DK&series_token=nRF9160',
+        },
+    },
+    PCA10100: {
+        name: 'nRF52833 DK',
+        cores: 1,
+        website: {
+            productPagePath: 'Development-Kits/nRF52833-DK',
+            buyOnlineParams: 'search_token=nRF52833-DK&series_token=nRF52833',
+        },
+    },
+    PCA10095: {
+        name: 'nRF5340 DK',
+        cores: 2,
+        website: {
+            productPagePath: 'Development-Kits/nRF5340-PDK',
+            buyOnlineParams: 'search_token=nRF5340-PDK&series_token=nRF5340',
+        },
+    },
+    PCA20020: {
+        name: 'Nordic Thingy:52',
+        cores: 1,
+        website: {
+            productPagePath: 'Prototyping-platforms/Nordic-Thingy-52',
+            buyOnlineParams: 'search_token=nRF6936&series_token=nRF52832',
+        },
+    },
+    pca20035: {
```

Comment:
I'd suggest to capitalize _pca20035_ key, and make sure the map is accessed by capitalized parameter.

Reply:
Already fixed in 261b9bd8f81583ce9dbe252e0a3703ca40088a0a in the device selector branch.